src/client.ts

```diff
@@ -0,0 +1,36 @@
+import axios, { InternalAxiosRequestConfig, AxiosResponse } from 'axios';
+import { logger } from './logger';
+
+const clientLogger = logger.child({}, { msgPrefix: '🌐 ' });
```

Comment:
Haha, what? I like it.

Reply:
Aha, great catch, @bickelj! It turns out error handling behaves differently between synchronously throwing an error and asynchronously returning a Promise that later rejects. I've added a commit to fix that!

Reply:
You should be able to see it with `LOG_LEVEL=debug`: `LOG_LEVEL=debug npm start -- candid lookup 12345678`.